requirements.txt

```diff
@@ -2,28 +2,28 @@ attrs==21.4.0
 autobahn==21.2.1
 Automat==20.2.0
 certifi==2021.10.8
-cffi==1.15.0
```

Comment:
These should be a separate review

Reply:
oops.  it was a separate review but hadn't been merged when this pr was submitted.